tests/test_getInstcal.py

```diff
@@ -61,7 +61,7 @@ def testInstcal(self):
         self.assertEqual(exp.getWidth(), self.size[0])
         self.assertEqual(exp.getHeight(), self.size[1])
         self.assertEqual(exp.getDetector().getId(), self.dataId["ccdnum"])
-        self.assertEqual(exp.getFilter().getFilterProperty().getName(), self.filter)
+        self.assertEqual(exp.getFilter().getName(), self.filter)
```

Comment:
As in lsst/pipe_tasks#428, I'm pretty sure this is an idiom for `getCanonicalName()` that predates its introduction.